assets/js/theme.js

```diff
@@ -211,7 +211,7 @@ var theme = function () {
         },
         // CountDown
         initCountDown: function () {
-            var austDay = new Date(2017,9,7);
+            var austDay = new Date(2018,8,29);
```

Comment:
Shouldn't it be `2018,8,29`? (the conference is taking place on _September_ 29th)

Reply:
In Javascript the months are between 0 and 11.

Reply:
Is the change that I made, isn't?

Reply:
so the 8th month is September? :thinking: I thought that was August :-D

Reply:
lol, it is, my bad :/ this just show how much Js I write in a daily basis

Reply:
My bad @archel. As @nubeiro I also haven't taken into account zero based JS months.